content/blog/moochris/2017-01-22_which_launcher.md

```diff
@@ -0,0 +1,25 @@
++++
+type = "blog"
+title = "Which Launcher?"
+author = "moochris"
+date = "2017-01-22 12:05:49+00:00"
+tags = ["haiku", "software"]
++++
+
+<a href="/files/blog/moochris/leafmenu.png" title="Leaf Menu"><img src="/files/blog/moochris/leafmenu.png" title="Leaf Menu" alt="Leaf Menu" align="right"></a>So you've installed Haiku from a recently nightly (or sometime soon, the R1 beta) and you're launching applications from the 'leaf' menu. Perfect, but there are a few more options to investigate if you want to quickly launch your favourite programs.
+
+<!--more-->
+
+The first option is the simplest. In the 'leaf' application menu, just click on the 'Applications' submenu (or any of the others) to open a tracker window with shortcuts. Right-click on a shortcut and in the context menu select 'Create link', then 'Desktop'. Hey presto, a shortcut icon on your desktop.
```

Comment:
Again, 'Deskbar menu' instead "'leaf' application menu". (Right-click-drag on Desktop is even easier, I think, but why not, for variety? :)